packages/react-components/src/util/toFormattedBalance.ts

```diff
@@ -53,11 +49,18 @@ const toFormattedBalance = (args: Arguments): string => {
   };
   const formattedBalance = formatBalance(balance, polkadotFormatBalanceOptions);
 
-  console.log('object', polkadotFormatBalanceOptions);
   const integerPart = formattedBalance.split('.')[0];
-  const decimalPart = balance.slice(-fixedPoint);
+  const decimalPart = balance.toString().slice(-fixedPoint);
 
   return `${integerPart}.${decimalPart}${unitPart}`;
 };
 
+const toFormattedBalance = (args: Arguments): string => {
+  if (BN.isBN(args.balance)) {
```

Comment:
This check seems to be unnecessary. In the definition of the 'Argument' interface, you are saying 'balance' is a BN.